content/en/blog/kernel-testing.md

```diff
@@ -0,0 +1,172 @@
+These tests should be executed automatically each time new code is integrated into our drivers, serving as a means to promptly identify any issue or flaw in the tested kernel versions.  
+With these objectives in mind, our approach should fulfill the following requirements:
+
+* Rapid and cost-effective VM creation: The process of creating these virtual machines should be efficient and budget-friendly.
+* Effortless distribution of VM images: We should ensure easy sharing and deployment of the virtual machine images.
+* Parallel execution of tests on multiple VMs: Tests should run concurrently on each virtual machine to expedite the process.
+* Reproducibility in local environments for debugging purposes: Developers should be able to replicate the test environment locally to investigate and troubleshoot issues.
+* Straightforward and user-friendly presentation of test results: The test results should be presented in a simple and intuitive manner to immediately spot failures.
+
+#### Ignite a Firecracker microVM
+
+[Weave Ignite](https://https://github.com/weaveworks/ignite) is used to provision the [firecracker](https://github.com/firecracker-microvm/firecracker) microVMs. Weave Ignite is an open-source tool designed for lightweight and fast virtual machine management. It enables users to effortlessly create and manage virtual machines (VMs) for various purposes, such as development, testing, and experimentation. 
+One of the main reasons why we chose to use this tool was its capability to create firecracker microVMs from kernels and rootfs packed as OCI images.
+Currently, we are using a patched version located at [a forked repository](https://github.com/therealbobo/ignite). These patches were essential to enable the booting of kernels that necessitated the use of an initrd (initial ramdisk).
+
+#### Kernel & Rootfs OCI images
+
+Virtual machines consist of two essential layers: the kernel and the rootfs. These layers are packaged and distributed as OCI (Open Container Initiative) images. The kernel image encompasses the kernel that the virtual machine relies on, in contrast the rootfs image serves as the fundamental building block of a virtual machine, offering the essential filesystem necessary for booting the VM. Typically, these rootfs images incorporate a Linux distribution. 
+For more info on how we build them please check the available [images documentation](https://github.com/falcosecurity/kernel-testing/tree/main/images).
+
+#### Ansible Playbooks
+
+Automation is accomplished through the utilization of [Ansible](https://docs.ansible.com/ansible/latest/index.html). A collection of [playbooks](https://github.com/falcosecurity/kernel-testing/tree/main/ansible-playbooks) is responsible for:
+
+* Orchestrating the provisioning of microVMs.
+* Configuring the machines.
+* Retrieving the code to be tested.
+* Eliminating the microVMs once the testing process is completed.
```

Comment:
I think that these are really small paragraphs, that do not distract that much; since we are presenting a deeply technical work, i don't think it hurts to leave them here!